mcc_codes.json

```diff
@@ -6255,13 +6255,22 @@
     "id": 694
   },
   {
+    "mcc": "3816",
+    "edited_description": "Home2Suites",
+    "combined_description": "",
+    "usda_description": "",
+    "irs_description": "",
+    "irs_reportable": "",
+    "id": 695
+  },
+  {
     "mcc": "3835",
     "edited_description": "* MASTERS ECONOMY INNS",
     "combined_description": "* MASTERS ECONOMY INNS",
     "usda_description": "* MASTERS ECONOMY INNS",
     "irs_description": "",
     "irs_reportable": "",
-    "id": 695
+    "id": 696
```

Comment:
Just the way the script is done https://github.com/clearhaus/mcc-codes/blob/master/csv-to-json.php#L52.

Not sure what the use case might've been originally since the ids can easily shift, and they will, as illustrated by this PR. :stuck_out_tongue: